searchlib/src/tests/attribute/document_weight_iterator/document_weight_iterator_test.cpp

```diff
@@ -56,47 +62,57 @@ void populate_string(AttributeVector::SP attr_ptr) {
     set_doc(attr, 7, "foo", 10);
 }
 
+template<CollectionType::Type CT>
 struct LongFixture {
     AttributeVector::SP attr;
     const IDocumentWeightAttribute *api;
-    LongFixture() : attr(make_attribute(BasicType::INT64, CollectionType::WSET, true)),
-                    api(attr->asDocumentWeightAttribute())
+    LongFixture()
+        : attr(make_attribute(BasicType::INT64, CollectionType::WSET, true)),
```

Comment:
Shouldn't we use CT instead of CollectionType::WSET?

Reply:
Yes

Reply:
Fixed in #27653